app/src/main/java/gateway/controller/CtrlAuthLogin.java

```diff
@@ -1,5 +1,6 @@
 package gateway.controller;
 import gateway.config.Config;
+import gateway.services.UtilValidator;
```

Comment:
Imported but not used. 

Use it to validate the request's arguments. For an example see

https://github.com/hawks-atlanta/gateway-java/blob/1970246448bf7ad98a09028f53c24694961439e5/app/src/main/java/gateway/controller/CtrlFileUpload.java#L27